src/main/java/seedu/address/logic/commands/DeleteContactFromTeamCommand.java

```diff
@@ -0,0 +1,80 @@
+package seedu.address.logic.commands;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.contact.Contact;
+import seedu.address.model.team.Team;
+
+import static java.util.Objects.requireNonNull;
+
+public class DeleteContactFromTeamCommand extends Command {
+
+        public static final String COMMAND_WORD = "delete-from-team";
+
+        public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes a contact from the team. "
+                + "Parameters: "
+                + "Team index number, contact index number \n"
+                + "Example: delete-from-team 2 3";
+
+        public static final String MESSAGE_SUCCESS = "Contact removed from team: %1$s";
+        public static final String MESSAGE_INEXISTENT_CONTACT_IN_TEAM = "A contact with this index does not exist in " +
+                "the team";
+
+        private final int teamIndex;
+        private final int contactIndex;
+
+        /**
+         * Creates a DeleteContactFromTeamCommand to delete the specified {@code Contact}
+         */
+        public DeleteContactFromTeamCommand(int teamIndex, int contactIndex) {
+            this.teamIndex = teamIndex - 1;
+            this.contactIndex = contactIndex - 1;
+        }
+
+        @Override
+        public CommandResult execute(Model model) throws CommandException {
+            requireNonNull(model);
+
+            if (teamIndex < 0 || teamIndex >= model.getCodeConnect().getTeamList().size()) {
+                throw new CommandException(Messages.MESSAGE_INVALID_TEAM_DISPLAYED_INDEX);
+            }
+
+            Team team = model.getCodeConnect().getTeamList().get(teamIndex);
+
+            if (contactIndex < 0 || contactIndex >= team.getMembers().size()) {
+                throw new CommandException(Messages.MESSAGE_INVALID_CONTACT_DISPLAYED_INDEX);
+            }
+            Contact contactToDelete = team.getMembers().get(contactIndex);
+
+            if (!team.hasMember(contactToDelete)) {
+                throw new CommandException(MESSAGE_INEXISTENT_CONTACT_IN_TEAM);
+            }
```

Comment:
This if statement might not be necessary, because if we did get the contact from the team's list we should already know that it has that member.